### ISSUE TITLE
Alpine image of server has problem with Web monitoring

### ISSUE THREAD

**Issue by ronnicek** [zabbix/zabbix-docker#141]
I've setup Alpine env for my monitoring and I found out some error in web testing. When I changed server for ubuntu everything is working well. So.. lets describe it :)

1. setup up alpine env
2. create host
3. prepare web scenario
4. as step load some page and check status code to 200
5. check graph after.. 2 - 3 hours :)

When I have Alpine as server I have following graph (showing hours as response?) - 
![zabbix_webtest](https://user-images.githubusercontent.com/3885042/30774544-90825214-a084-11e7-83f4-dc540b9325c1.png)

But when I changed server to Ubuntu, everything is normal and working well:
![ubuntu_webtest](https://user-images.githubusercontent.com/3885042/30774555-ba265868-a084-11e7-97b2-d4540d453ac4.png).

I wrote this also to Zabbix Forum, but I think here is the right place to put it. I dont know how Zabbix is checking response from the server, but Alpine has some things different which I find out when I want to put there also SSL checking (date is from busybox [this can be solved by installing coreutils], time command has a bit different output also). So maybe its problem with some parsing somewhere?

**Comment by ronnicek** [comment]
And maybe also this image will help (you can see, that peaks in response times has inverse peaks in speed graph when it was on Alpine..
![speed_response](https://user-images.githubusercontent.com/3885042/30774702-13aa1364-a087-11e7-8f4b-9a78ca7d9833.png)

**Comment by ronnicek** [comment]
And also here are responses from docker host:
```
Sat Sep 23 12:45:20 CEST 2017: 0m0.627s
Sat Sep 23 12:50:01 CEST 2017: 0m0.664s
Sat Sep 23 12:55:01 CEST 2017: 0m0.690s
Sat Sep 23 13:00:01 CEST 2017: 0m0.629s
Sat Sep 23 13:05:01 CEST 2017: 0m0.612s
Sat Sep 23 13:10:01 CEST 2017: 0m0.643s
Sat Sep 23 13:15:01 CEST 2017: 0m0.652s
Sat Sep 23 13:20:01 CEST 2017: 0m0.617s
Sat Sep 23 13:25:01 CEST 2017: 0m0.638s
Sat Sep 23 13:30:01 CEST 2017: 0m0.631s
Sat Sep 23 13:35:01 CEST 2017: 0m0.634s
Sat Sep 23 13:40:01 CEST 2017: 0m0.838s
Sat Sep 23 13:45:01 CEST 2017: 0m0.656s
Sat Sep 23 13:50:01 CEST 2017: 0m0.651s
```

And here are responses from docker image:
```
Sat Sep 23 12:45:20 CEST 2017: 0m0.71s
Sat Sep 23 12:50:01 CEST 2017: 0m0.73s
Sat Sep 23 12:55:01 CEST 2017: 0m0.87s
Sat Sep 23 13:00:01 CEST 2017: 0m0.70s
Sat Sep 23 13:05:01 CEST 2017: 0m0.98s
Sat Sep 23 13:10:01 CEST 2017: 0m0.65s
Sat Sep 23 13:15:01 CEST 2017: 0m0.69s
Sat Sep 23 13:20:01 CEST 2017: 0m0.71s
Sat Sep 23 13:25:01 CEST 2017: 0m0.75s
Sat Sep 23 13:30:01 CEST 2017: 0m0.65s
Sat Sep 23 13:35:01 CEST 2017: 0m0.68s
Sat Sep 23 13:40:01 CEST 2017: 0m0.76s
Sat Sep 23 13:45:01 CEST 2017: 0m0.75s
Sat Sep 23 13:50:01 CEST 2017: 0m0.70s
```

(here I find out that time on Alpine is a bit different and there is also space between 0m and 0.70s in output)

**Comment by MattParr** [comment]
I'm also seeing this.
Here are some reported response times from 500 latest values:

```
2017-10-02 21:11:25 | 4295.8685
2017-10-02 21:10:24 | 4295.8697
2017-10-02 21:09:23 | 4296.2363
2017-10-02 21:08:22 | 1.2476
2017-10-02 21:07:20 | 1.0568
2017-10-02 21:06:19 | 1.0922
2017-10-02 21:05:18 | 4295.9191
2017-10-02 21:04:17 | 4295.8959
2017-10-02 21:03:16 | 4296.5499
2017-10-02 21:02:15 | 4295.7312
```

and another site from another host:

```
2017-10-02 21:04:17 | 4295.8959
2017-10-02 21:03:16 | 4296.5499
2017-10-02 21:02:15 | 4295.7312
2017-10-02 21:01:14 | 1.433
2017-10-02 21:00:12 | 1.0947
2017-10-02 20:59:11 | 4296.3882
2017-10-02 20:58:10 | 1.1118
2017-10-02 20:57:09 | 1.2529
2017-10-02 20:56:08 | 4295.946
2017-10-02 20:55:07 | 1.2193
2017-10-02 20:54:06 | 4296.253
```

**Comment by MattParr** [comment]
This issue for libcurl is the culprit.
https://github.com/curl/curl/issues/1769

**Comment by ronnicek** [comment]
Hi,

I checked 3.4 repository which is used in Docker files for Zabbix and it should be there as well.

https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.4&repo=&arch=&maintainer=

**Comment by MattParr** [comment]
I believe the fix is in 7.55.1

**Comment by dotneft** [comment]
Unfortunately we can not use alpine 3.6, because there is different library, libressl is not supported by Zabbix. Maybe in future...

Also Matt supposed the issue has been fixed. ronnicek, could you check the issue?

**Comment by ronnicek** [comment]
It's not fixed because in Alpine is only 7.55.0 :)
